UnitTestParking/UnitTestParking.cs

```diff
@@ -9,31 +9,68 @@ public class UnitTestParking
         [Fact]
         public void isParkingPossible_WhenSpaceAvailable_ReturnsTrue()
         {
-            var parkingLot = new ParkingLotService(1);
+            var accounts = new BankAccountService();
```

Comment:
organise each test in 3 categories, Arrange, Act, Assert. So extract the act from inside the Assert.True and use comments like this //Arrange then next block with //Act wich will be the method you test and then all the //Asserts. 

Also the name of the service you test it should be sut (instead of accounts) which means system under test